packages/core/src/components/modal/modal.scss

```diff
@@ -1,12 +1,12 @@
 @import '~@atomium/scss-utils/index';
 
 :host {
-  --atom-modal-z-index: var(--zindex-1000);
+  --atom-modal-zindex: var(--zindex-1000);
```

Comment:
we need to update docs

Reply:
even if it's a local variable? i've followed the atom-tag example and it didn't had a example on docs :thinking:

Reply:
[like a select example](https://juntossomosmais.github.io/atomium/?path=/docs/components-select--docs)

<img width="1019" alt="image" src="https://github.com/user-attachments/assets/b4f6eb29-24a7-4b07-99e4-95036f805a72">

Reply:
done it here!! [5efe8ab](https://github.com/juntossomosmais/atomium/pull/617/commits/5efe8abba05d32cdea8b157d550e546ffdf78bb1)

Reply:
thanks for the example